src/shapes.ts

```diff
@@ -337,14 +353,11 @@ export class Text extends Shape {
     return this.splitInWords();
   }
 
-  private pushSeparatorInWords(pickedChars: number): [string, number] {
-    return [this.text[pickedChars], pickedChars + 1]
-  }
-
   private buildWord(pickedChars: number, word: string): [string, number] {
```

Comment:
In the img diff, it looks like the linebreaks don't occur at the same place, and sometimes at the middle of a word (separated by a "_" for ex). I guess this comes from this change. Is it the wanted  behavior ?

Reply:
Yes it is wanted because '_' is a separator but I'm open to other opinions :)

Reply:
No, it's all fine to me. I just wanted to be sure it wasn't a bug introduced